machine.md

```diff
@@ -252,6 +252,33 @@ spec:
       persistentVolumeClaim:
         claimname: myclaim
 ```
+## Resources Requests and Limits
```

Comment:
Can you also add a link to the k8s documentation, so that people will have a little bit more context on how to use these fields?